src/utils/makeParentChildTree.ts

```diff
@@ -79,3 +79,11 @@ export function iterateAllChildren<T> (item: TreeItem<T>, callback: Function): v
     })
   }
 }
+
+export function recursiveSortChildren<T> (item: TreeItem<T>, sortCallback: (a: TreeItem<T>, b: TreeItem<T>) => number) {
+  const children = item[CHILDREN]
+  if (children) {
+    item[CHILDREN] = children.sort(sortCallback)
```

Comment:
sorting is in place, so I don't think the assignment here has an effect (you're assigning the array to itself)
if you want to change the reference, you probably need `item[CHILDREN] = [...children.sort(sortCallback)]`